dev-libs/mongo-c-driver/mongo-c-driver-1.30.6.ebuild

```
@@ -0,0 +1,201 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Generate using https://github.com/thesamesam/sam-gentoo-scripts/blob/main/niche/generate-cmake-docs as a template
+# Set to 1 if prebuilt, 0 if not
+# (the construct below is to allow overriding from env for script)
+: ${MONGO_C_DRIVER_DOCS_PREBUILT:=1}
+
+# Default to generating man pages if no prebuilt; overridden later
+MONGO_C_DRIVER_DOCS_USEFLAG="+man"
+
+VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/mongo-c-driver.asc
+
+inherit cmake dot-a verify-sig
+
+DESCRIPTION="Client library written in C for MongoDB"
+HOMEPAGE="https://github.com/mongodb/mongo-c-driver"
+SRC_URI="
+	https://github.com/mongodb/mongo-c-driver/releases/download/${PV}/${P}.tar.gz
+	verify-sig? (
+		https://github.com/mongodb/mongo-c-driver/releases/download/${PV}/mongo-c-driver-${PV}.tar.gz.asc
+	)
+"
+
+if [[ ${MONGO_C_DRIVER_DOCS_PREBUILT} == 1 ]] ; then
+	SRC_URI+=" !man? ( https://people.znc.in/~dessa/gentoo/distfiles/${CATEGORY}/${PN}/${PN}-${PV}-docs.tar.xz )"
+	MONGO_C_DRIVER_DOCS_USEFLAG="man"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm64 ~hppa ~riscv ~x86"
+IUSE="debug examples ${MONGO_C_DRIVER_DOCS_USEFLAG} sasl +ssl static-libs test +test-full"
+REQUIRED_USE="
+	test? ( static-libs )"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-arch/snappy:=
+	app-arch/zstd:=
+	~dev-libs/libbson-${PV}[static-libs?]
```

```suggestion
	~dev-libs/libbson-${PV}[static-libs?]
	dev-libs/libutf8proc:=
```

```
 * VDB: detected possibly incorrect RDEPEND (dev-libs/mongo-c-driver-1.30.6)
 *  > dev-libs/libutf8proc:=
```

https://github.com/mongodb/mongo-c-driver/blob/1.30.6/src/libmongoc/CMakeLists.txt#L812
https://github.com/mongodb/mongo-c-driver/commit/47dc666478cfdde9f38ccefab033c6e0865e3fbf